src/graphql/data/nft/Details.ts

```diff
@@ -107,7 +108,7 @@ export function useDetailsQuery(address: string, tokenId: string): [GenieAsset,
       collectionSymbol: asset?.collection?.image?.url,
       imageUrl: asset?.image?.url,
       animationUrl: asset?.animationUrl ?? undefined,
-      marketplace: asset?.listings?.edges[0]?.node.marketplace.toLowerCase() as any,
+      marketplace: listing?.marketplace.toLowerCase() as any,
```

Comment:
can we update this to be casted to Markets instead of any?

Reply:
`listing.marketplace` is `NftMarketplace` (`src/graphql/data/nft/__generated__/DetailsQuery.graphql.ts#13`)

the return type on this function is `[GenieAsset, CollectionInfoForAsset] | undefined`

`GenieAsset` wants a `marketplace` type of `Markets` (`src/nft/types/common/common.ts#138`)

I fiddled with a way to cast this, but I'm not comfortable enough with the nft codebase yet to feel confident about a change here. `as unknown as Market` would work, but the actual types are pretty different. I'd feel more comfortable (sans context) actually doing a refactor to achieve some behavior we want. If you know for sure what will work well given downstream consumption expectations, I'm happy to make the change for ya here. 

### context:
```
export type NftMarketplace = "CRYPTOPUNKS" | "FOUNDATION" | "LOOKSRARE" | "NFT20" | "NFTX" | "OPENSEA" | "SUDOSWAP" | "X2Y2" | "%future added value";
```
vs 
```
export enum Markets {
  LooksRare = 'looksrare',
  X2Y2 = 'x2y2',
  NFT20 = 'nft20',
  NFTX = 'nftx',
  Opensea = 'opensea',
  Rarible = 'rarible',
  Uniswap = 'Uniswap',
  Uniswap_V2 = 'Uniswap_V2',
  SushiSwap = 'SushiSwap',
  SuperRare = 'superrare',
  KnownOrigin = 'knownorigin',
  WETH = 'weth',
  Cryptopunks = 'cryptopunks',
  CryptoPhunks = 'cryptophunks',
}```